src/main/java/com/codeborne/selenide/impl/WebElementSelector.java

```diff
@@ -66,11 +67,11 @@ protected Boolean sizzleLoaded() {
       return false;
     }
   }
-  
+
   protected synchronized void injectSizzle() {
     if (sizzleSource == null) {
       try {
-        sizzleSource = IOUtils.toString(currentThread().getContextClassLoader().getResource("sizzle.js"));
+        sizzleSource = IOUtils.toString(currentThread().getContextClassLoader().getResource("sizzle.js"), Charset.defaultCharset());
```

Comment:
It's a bad idea to rely on default encoding. It can be `cp-1251`, `KOI8R`, Chinese - whatever else.

Use `StandardCharsets.UTF8` instead.